R/tokens_group.R

```diff
@@ -0,0 +1,15 @@
+#' recombine documents tokens by groups
+#' 
+#' @param x \link{tokens} object
```

Comment:
You can ignore these comments if you `tokens_group` is not exported, but still it would be nice for this to work the same way as `dfm_group`.